### PR TITLE
error handler for deleteById and findById

### PR DESCRIPTION
**Pull request by RomanukAlex**

### CODE REVIEW

File: rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+import com.epam.repair.exception.OrderNotFoundException;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@ControllerAdvice
+public class errorHandler {
```
Comment (aleksieiev) [inline rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java]: `errorHandler`
Are you seriously?

File: rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    @ExceptionHandler(OrderNotFoundException.class)
+    public void springHandleNotFound(HttpServletResponse response) throws IOException {
```
Comment (aleksieiev) [inline rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java]: `springHandleNotFound`
bad name of the method

File: service/src/main/java/com/epam/repair/exception/OrderNotFoundException.java [hunk 3]
```diff
@@ -0,0 +1,9 @@
+package com.epam.repair.exception;
+
+public class OrderNotFoundException extends RuntimeException {
```
Comment (aleksieiev) [inline service/src/main/java/com/epam/repair/exception/OrderNotFoundException.java:6]: JavaDoc?

File: service/src/main/java/com/epam/repair/service/RepairOrderServiceImpl.java [hunk 4]
```diff
@@ -51,11 +50,11 @@ public RepairOrder update(RepairOrder repairOrder) {
     }
 
     @Override
-    public void deleteById(Integer repairOrderId) {
+    public void deleteById (Integer repairOrderId) {
         if (repairOrderDao.existsById(repairOrderId)) {
             repairOrderDao.deleteById(repairOrderId);
```
Comment (aleksieiev) [inline service/src/main/java/com/epam/repair/service/RepairOrderServiceImpl.java]: What if somewhere between these operations connection to DB will be lost? Or another thread do the same operation.

File: rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java (hunk 1 above)
Comment (RomanukAlex) [reply rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java]: Sorry...

Comment (aleksieiev) [reply rest-app/src/main/java/com/epam/repair/rest/handlers/errorHandler.java]: ^
||
It's also bad name for class ;-)

**Comment by aleksieiev** [comment]
Bad commit naming: `fix`
What kind of fix? What should it fix? What happens if I apply it on production?

**Review by sergeyarhutik — APPROVED** [review]

**Pull request closed without merging**